README.rst

```diff
@@ -19,14 +20,16 @@ Just install via ``pip``:
 
    $ pip install sphinxcontrib-svg2pdfconverter
 
-You can choose between Inkscape and libRSVG by either adding
-``sphinxcontrib.inkscapeconverter`` or ``sphinxcontrib.rsvgconverter`` to the
-``extensions`` list in your ``conf.py``.
+You can choose between Inkscape, libRSVG and cairosvg by adding
+``sphinxcontrib.inkscapeconverter``, ``sphinxcontrib.rsvgconverter`` or
+``sphinxcontrib.cairosvgconverter`` to the ``extensions`` list in your
+``conf.py``.
 
 Make sure to have either ``inkscape`` or the ``rsvg-convert`` command available
 in your systems ``PATH`` and, if necessary, adapt the
 ``inkscape_converter_bin`` or ``rsvg_converter_bin`` config value respectively.
 
+``cairosvg`` requires ``libcairo-2`` to work.
```

Comment:
According to the [Installation section of the CairoSVG documentation](https://cairosvg.org/documentation/), it is more complicated than that. We should probably just write that one has to install CairoSVG using their distributions package manager or according to the CairoSVG documentation.